ThingIFSDK/ThingIFSDK/IoTOperations/IoTRequestOperation.swift

```diff
@@ -13,7 +13,9 @@ internal enum HTTPMethod: String {
     case GET = "GET-deprecated"
     case get = "GET"
     case post = "POST"
-    case PUT = "PUT"
+    @available(iOS, deprecated: 1.0, message: "use put")
+    case PUT = "PUT-deprecated"
```

Comment:
@yongpingchen 

If we replace All `PUT` to `put` in this PR, we must make some unrelated fixes for this PR. I think it is not good PR which includes unrelated fixes for the purpose of the PR.

XCode points out the code using `PUT` with `@availale` annotation. I will replace them when needed.